fabcar/startFabric.sh

```diff
@@ -109,8 +109,25 @@ Go:
   Start by changing into the "go" directory:
     cd go
 
+  Next, set hosts:
+    sudo vim /etc/hosts
+
+  add Under the content into  /etc/hosts
+    ip	peer0.org1.example.com  
+    ip	peer0.org2.example.com  
+    ip	peer0.org2.example.com  
+
+    for example:
+    127.0.0.1       peer0.org1.example.com  peer0.org2.example.com  orderer.example.com
+
+  Next, set environment variable:
+    export DISCOVERY_AS_LOCALHOST=true
+
   Then, install dependencies and run the test using:
-    go run fabcar.go
+    1. export DISCOVERY_AS_LOCALHOST=true
```

Comment:
thanks for adding above step to this documentation.

My opinion is that the you made documentation in this script fairly clear, and we do not need alternate script `./runfabcar.sh`. So your instruction on setting env var is sufficient

Reply:
Personally I prefer to have scripts running everything like this. The first thing I did when started looking into running test-network and starting all services was to create script with all of the steps from documentation so other team members don't need to go through it again.